src/types/decimal/mod.rs

```diff
@@ -14,7 +14,31 @@ use std::convert::{TryFrom, TryInto};
 use std::fmt::{Display, Formatter};
 use std::ops::Neg;
 
+pub(crate) mod coefficient;
+mod magnitude;
+
 /// An arbitrary-precision Decimal type with a distinct representation of negative zero (`-0`).
```

Comment:
Is it worth including some comment about what negative zero is for, what it means?